### PR TITLE
Homepage Registration and Reporting Section

### PR DESCRIPTION
**Pull request by xtine**
Issue #644 

Relies on: https://github.com/18F/fec-style/pull/585

screen shot 2016-12-14 at 10 12 46 pm
screen shot 2016-12-14 at 10 13 14 pm

### CODE REVIEW

File: fec/home/templates/home/home_page.html [hunk 1]
```diff
@@ -33,6 +33,51 @@
...
+              <a href="/registration-and-reporting/request-additional-information">Learn how to respond »</a>
+            </div>
+          </div>
+        </div>
+        <div class="grid__item">
+          <div class="icon-heading">
+            <div class="icon-heading--calendar-circle"></div>
+            <div class="icon-heading__content">
+              <h3>Candidates and committees must file regular reports about their financial activity</h3>
+              <a href=" ">Find upcoming reporting deadlines »</a>
```
Comment (emileighoutlaw) [inline fec/home/templates/home/home_page.html]: This link should point to the calendar: https://beta.fec.gov/calendar/ right?

**Comment by emileighoutlaw** [comment]
Pushed a couple small copyedits, but this gets a content thumbs up from me. Thanks, Christine!

File: fec/home/templates/home/home_page.html (hunk 1 above)
Comment (xtine) [inline fec/home/templates/home/home_page.html]: It should point to the calendar, but with filing deadlines applied. The calendar filters are fixed in dev but not prod yet. The same is for the other unfilled link that needs calendar with outreach category applied.

File: fec/home/templates/home/home_page.html [hunk 2]
```diff
@@ -33,6 +33,51 @@
...
+              <a href="/registration-and-reporting/request-additional-information">Learn how to respond »</a>
+            </div>
+          </div>
+        </div>
+        <div class="grid__item">
+          <div class="icon-heading">
+            <div class="icon-heading--calendar-circle"></div>
+            <div class="icon-heading__content">
+              <h3>Candidates and committees must file regular reports about their financial activity</h3>
+              <a href="/calendar">Find upcoming reporting deadlines »</a>
```
Comment (noahmanger) [inline fec/home/templates/home/home_page.html]: This should have filters applied. I think what we want is: 
`calendar/?category=EC+Periods&category=FEA+Periods&category=IE+Periods&category=report-E&category=report-M&category=report-Q`

File: fec/home/templates/home/home_page.html [hunk 3]
```diff
@@ -33,6 +33,51 @@
...
+            </div>
+          </div>
+        </div>
+        <div class="grid__item">
+          <div class="icon-heading">
+            <div class="icon-heading--training-circle"></div>
+            <div class="icon-heading__content">
+              <h3>The FEC regularly hosts training events for candidates and committees</h3>
+              <a href="/calendar">Find upcoming webinars, conferences and trainings »</a>
```
Comment (noahmanger) [inline fec/home/templates/home/home_page.html]: And this should have these filters:

`/calendar/?category=Conferences&category=Roundtables`

File: fec/home/templates/home/home_page.html [hunk 4]
```diff
@@ -33,6 +33,51 @@
...
   </section>
+
+  <section class="slab slab--secondary">
+    <div class="container">
+      <h1><a href="/registration-and-reporting">Registration and reporting</a></h1>
+
+      <div class="grid grid--2-wide">
+        <div class="grid__item">
+          <div class="icon-heading">
+            <div class="icon-heading--checklist-circle"></div>
```
Comment (noahmanger) [inline fec/home/templates/home/home_page.html]: Minor thing (and my fault for not documenting), but `.icon-heading--checklist-circle` should actually just be on `.icon-heading`. The whole thing works without needing an extra empty div.

Comment (xtine) [inline fec/home/templates/home/home_page.html]: Nice. I was looking at pattern library for example, so I'll update it with another icon heading example.

**Pull request merged**